pkg/relay/ethereum/tx.go

```diff
@@ -113,7 +114,7 @@ func (c *Chain) GetMsgResult(id core.MsgID) (core.MsgResult, error) {
 	if receipt.Status == gethtypes.ReceiptStatusSuccessful {
 		return c.makeMsgResultFromReceipt(&receipt.Receipt, "")
 	}
-	revertReason, err := c.getRevertReasonFromReceipt(ctx, receipt)
+	revertReason, _, err := c.getRevertReasonFromReceipt(ctx, receipt)
 	if err != nil {
 		logger.Error("failed to get revert reason", err)
```

Comment:
Should we also contain `errorData` in the error log?

Reply:
🤔 ... sure. I will fix it.

Reply:
@bluele fixed https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/39/commits/fcb8da1647c8489d0c8cb2e5fdf6d8f66beea6ab

Reply:
ah ... should we also add tx_hash here?

Reply:
added https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/39/commits/99492cdacef7cee5d5273bafe195f3a11f16d53e